app/src/main/java/com/arctouch/codechallenge/view/home/HomeAdapter.kt

```diff
@@ -33,6 +32,7 @@ class HomeAdapter: PagedListAdapter<Movie, HomeAdapter.ViewHolder>(movieDiffUtil
         private val movieImageUrlBuilder = MovieImageUrlBuilder()
 
         fun bind(movie: Movie) {
+            movie!!.genres = Cache.genres.filter { movie!!.genreIds?.contains(it.id) == true }
```

Comment:
Here `movie` is no longer nullable, so the (ugly) `!!` is no longer necessary ...